gemma-teixido/pre-curso/tema-2/airlines-pro.js

```diff
@@ -0,0 +1,407 @@
+/*
+
+1. ADMIN or USER
+ 1.1. Si eres admin -> a) crear más vuelos (via prompt) && ! + de 15 vuelos -> alert().
+ b) poder elminar vuelos con el ID
+
+ 1.2. Si eres user -> a) buscar por precio (más alto, más bajo o igual) + 
+ despedida ("Gracias por su compra, vuelva pronto.")
+*/
+
+let flights = [
+  { id: 0, to: "Bilbao", from: "Barcelona", cost: 1600, scale: false },
+
+  { id: 1, to: "New York", from: "Barcelona", cost: 700, scale: false },
+
+  { id: 2, to: "Los Angeles", from: "Madrid", cost: 1100, scale: true },
+
+  { id: 3, to: "Paris", from: "Barcelona", cost: 210, scale: false },
+
+  { id: 4, to: "Roma", from: "Barcelona", cost: 150, scale: false },
+
+  { id: 5, to: "London", from: "Madrid", cost: 200, scale: false },
+
+  { id: 6, to: "Madrid", from: "Barcelona", cost: 90, scale: false },
+
+  { id: 7, to: "Tokyo", from: "Madrid", cost: 1500, scale: true },
+
+  { id: 8, to: "Shangai", from: "Barcelona", cost: 800, scale: true },
+
+  { id: 9, to: "Sydney", from: "Barcelona", cost: 150, scale: true },
+
```

Comment:
Remove break points number 13, 15, 17, 19, 21, 23, 25, 27, 29 and 31. Unified code style.